packages/infra/lib/ihe-stack.ts

```diff
@@ -183,6 +187,43 @@ export class IHEStack extends Stack {
       secrets,
       posthogSecretName,
       alarmSnsAction,
+      name: "IHEInboundPatientDiscovery",
+      entry: "ihe-inbound-patient-discovery",
+    });
+
+    const patientDiscoveryLambdaV2 = this.setupPatientDiscoveryLambda({
+      props,
+      lambdaLayers,
+      vpc,
+      secrets,
+      posthogSecretName,
+      alarmSnsAction,
+      name: "IHEInboundPatientDiscoveryV2",
+      entry: "ihe-gateway-v2-inbound-patient-discovery",
```

Comment:
nit: I imagine these are optional until we remove the old code; could make the values for the new ones the default of the parameter so we don't need to change this code afterwards. But defs NABD if we revert this later to not pass these as parameters once we don't have the v1 anymore.

Reply:
I will change this in the downstream PR to not pass as parameters